app/views/users/show.html.erb

```diff
@@ -2,6 +2,19 @@
   <p id="notice"><%= notice %></p>
 <% end %>
 
+<div class="search-bar">
+  <div class="search-form">
+    <%= form_with(url: user_path(@user.uid), method: "get") do %>
+      <%= text_field_tag(:q, @search_terms, {placeholder: "Search by DOI, Title, Name, ARK"}) %>
+      <%= submit_tag("Search", class: "btn btn-primary") %>
+    <% end %>
+
+    <% if @works_found == 0 && @search_terms.present? %>
+      <p>No works found for <i><%= @search_terms %></i></p>
+    <% end %>
```

Comment:
In UI terms, slight preference to put the "no results" message where the results would be, if there were any.

If this were an application-wide search, then up top makes sense, but otherwise better to place to the search box right above the results.

But just my opinion!

Reply:
Good idea. Done.